runhouse/resources/hardware/cluster.py

```diff
@@ -158,6 +158,7 @@ def config_for_rns(self):
             config["ssh_creds"] = self.ssh_creds()
         else:
             config["ips"] = [self.address]
+            config["ssh_creds"] = None
```

Comment:
I don't believe this is correct. Why should ssh_creds be None-d out?

Reply:
There's a basic resource reconstruction test, which uses the Cluster(**config) to reconstruct a cluster. In that, ssh_creds is None'd out. This has to align exactly with `config_for_rns` for the test to pass, and so instead of it being unset, I've set it to None. Normally it would just not exist in the dict